client/app/src/main/java/com/example/criengine/MainActivity.java

```diff
@@ -9,12 +9,24 @@
 
 public class MainActivity extends AppCompatActivity {
     private Button myProfileActivityButton;
+    private Button requestActivityButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+
+        requestActivityButton = findViewById(R.id.requested_books_activity);

```

Comment:
I don't think we want these MainActivity changes in the final PR since the actual main activity will be different

Reply:
> 
> 
> Yeah, that is kinda what I was afraid of haha. It looks pretty wack but its the best that we can really do.
> 
> Can you center the Title and try to bring the "Rejected" away from the left (try to center that as well if possible).
> An extra thing we could try to do is if a book is rejected, try to color that text in dark red while if it were accepted, then color it in dark green.

We don't have navigation yet, and it doesn't seem like we will until next sprint. I figure this will be an easy way to jump between pages for everyone until nav is done.
And we can always update the `AndroidManifest.xml` to change the starting activity

Reply:
Ah yup, that make sense then